platform/debian/kanidmd/rules

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+
+# be REALLY noisy
+export DH_VERBOSE=1
+export DH_OPTIONS=-v
+
+export DEB_BUILD_OPTIONS=noautodbgsym
+
+export DEB_BUILD_OPTIONS=noautodbgsym
+PACKAGE=kanidmd
+PKGDIR=debian/${PACKAGE}
+BINDIR=${PKGDIR}/usr/sbin/
+SHARED_DIR=${PKGDIR}/usr/share/${PACKAGE}
+
+%:
+	dh $@
+
+override_dh_auto_clean:
+# cargo clean
+
+override_dh_autoreconf:
+
+override_dh_auto_build:
+	KANIDM_BUILD_PROFILE=release_suse_generic dh_auto_build -- release/kanidmd
```

Comment:
Should be set on all the make release/* targets

Reply:
Do you mean in the debian `rules` files so the CPU flags are set?

Reply:
Every call to `dh_auto_build -- release/target` needs the profile set in all the various packages, because as you say, it affects cpu flags and potentially in the future, other values.

Reply:
Cool, just confirming that's what you mean, my inferring_meaning engine is a bit broken lately.